graylog2-web-interface/src/views/components/aggregationwizard/FieldSelect.tsx

```diff
@@ -115,25 +130,39 @@ const FieldSelect = ({
     .toArray()
     .sort(sortByLabel), [activeQuery, excludedFields, fieldTypes.queryFields, isFieldQualified]);
 
+  const _onSelectAllRest = useCallback(() => onSelectAllRest(fieldOptions.map(({ value: fieldValue }) => fieldValue)), [fieldOptions, onSelectAllRest]);
+
+  const _showSelectAllRest = !!fieldOptions?.length && showSelectAllRest && typeof _onSelectAllRest === 'function';
+
+  const _showDeSelectAll = showDeSelectAll && typeof onDeSelectAll === 'function';
+
   return (
-    <Select options={fieldOptions}
-            inputId={`select-${id}`}
-            forwardedRef={selectRef}
-            allowCreate={allowCreate}
-            className={className}
-            onMenuClose={onMenuClose}
-            openMenuOnFocus={openMenuOnFocus}
-            persistSelection={persistSelection}
-            clearable={clearable}
-            placeholder={placeholder}
-            name={name}
-            value={value}
-            aria-label={ariaLabel}
-            optionRenderer={OptionRenderer}
-            size={size}
-            autoFocus={autoFocus}
-            menuPortalTarget={menuPortalTarget}
-            onChange={onChange} />
+    <>
+      <Select options={fieldOptions}
+              inputId={`select-${id}`}
+              forwardedRef={selectRef}
+              allowCreate={allowCreate}
+              className={className}
+              onMenuClose={onMenuClose}
+              openMenuOnFocus={openMenuOnFocus}
+              persistSelection={persistSelection}
+              clearable={clearable}
+              placeholder={placeholder}
+              name={name}
+              value={value}
+              aria-label={ariaLabel}
+              optionRenderer={OptionRenderer}
+              size={size}
+              autoFocus={autoFocus}
+              menuPortalTarget={menuPortalTarget}
+              onChange={onChange} />
+      {(_showSelectAllRest || _showDeSelectAll) && (
+      <ButtonRow>
+        {_showSelectAllRest && <Button bsSize="xs" onClick={_onSelectAllRest}>Select all rest fields</Button>}
```

Comment:
Imo we can just call it "Select all fields". For a user there is no difference between "Select all fields" and "Select all rest fields".